nav2_controller/README.md

```diff
@@ -0,0 +1,11 @@
+# Nav2 Controller
+
+The Nav2 Controller is an [Execution Module](../doc/requirements/requirements.md) that implements the `nav2_behavior_tree::FollowPathAction` interface.
```

Comment:
Maybe remove the reference to behavior tree and intead mention it implements a `nav2_msgs::action::FollowPath` action server.